index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <h4>My Logo</h4>
+        <nav>
+            <a href="">
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
+        </nav>
+    </header>
+
+    <main>
+        <section class="Profile">
+                <h1>I’m Temesgen <br> Glad to see you!</h1>
+                <p>
```

Comment:
As required by the previous reviewer, please update the design to stick to Figma.

Please update the size and the color
Your design:

<img width="410" alt="image" src="https://user-images.githubusercontent.com/7243132/170825555-fc097119-24da-425d-bda3-1d6d738bb126.png">

Figma
<img width="537" alt="image" src="https://user-images.githubusercontent.com/7243132/170825575-63ca5f23-6102-40b7-b31f-30dd56e8a7c7.png">